source/trx2junit/Program.cs

```diff
@@ -39,22 +40,24 @@ private static async Task RunAsync(string[] args)
             Thread.CurrentThread.CurrentCulture   = CultureInfo.InvariantCulture;
             Thread.CurrentThread.CurrentUICulture = CultureInfo.InvariantCulture;
 
-            string trxFile   = args[0];
-            string jUnitFile = Path.ChangeExtension(trxFile, "xml");
-
-            Console.WriteLine($"Converting trx-file '{trxFile}' to JUnit-xml...");
+            Console.WriteLine($"Converting {args.Length } trx file(s) to JUnit-xml...");
             DateTime start = DateTime.Now;
 
-            var utf8 = new UTF8Encoding(false);
+            await Task.WhenAll(args.Select(Convert));
```

Comment:
Does this work with `trx2junit results/*.trx` as allowed in the description?
I.e. is there every file in the path transformed.

Maybe there should be a test covering the scenarios for the argumentes given.
If you won't add one, I'll add the tests after this PR is done.

Reply:
it works in bash, because the wildcard gets expanded by the shell before calling the program. I think it'd be non-trivial to add wildcard support in C#, maybe just update the docs to specify that the example only works on shells that support wildcards?

Reply:
Updated docs to be clearer about the shell handling the wildcards.

I think https://www.nuget.org/packages/Microsoft.Extensions.FileSystemGlobbing could be a solution for handling older shells, but my needs are met by this implementation so I'm unlikely to expand in that direction.

Reply:
For your info: https://github.com/gfoidl/trx2junit/pull/17 adds some tests for this.